Resources/public/js/views/services/plugins/ez-editcontentplugin.js

```diff
@@ -0,0 +1,113 @@
+/*
+ * Copyright (C) eZ Systems AS. All rights reserved.
+ * For full copyright and license information view LICENSE file distributed with this source code.
+ */
+YUI.add('ez-editcontentplugin', function (Y) {
+    "use strict";
+    /**
+     * Provides the edit content plugin
+     *
+     * @module ez-editcontentplugin
+     */
+    Y.namespace('eZ.Plugin');
+
+    /**
+     * Object user load plugin. It sets an event handler to load contents
+     * in an object relation list field.
+     * //TODO update
+     * In order to use it you need to fire the `loadUser` event with two parameters:
+     *  - `userId` of the user to be loaded
+     *  - `attributeName` where to store the result
+     *
+     * @namespace eZ.Plugin
+     * @class UserLoad
+     * @constructor
+     * @extends eZ.Plugin.ViewServiceBase
+     */
+    Y.eZ.Plugin.EditContent = Y.Base.create('editContentplugin', Y.eZ.Plugin.ViewServiceBase, [], {
+
+        initializer: function () {
+            this.onHostEvent('*:editContent', this._editContent);
+        },
+
+        /**
+         * editAction event handler, makes the application navigate to edit the
+         * content available in the event facade
+         *
+         * @method _editContent
+         * @protected
+         * @param {Object} e event facade of the editAction event
+         * //TODO need content
+         * //TODO need languageCode
+         */
+        _editContent: function (e) {
```

Comment:
would better name something like `_checkExistingDraft` to reflect what it actually does